### PR TITLE
Add video process test case

### PR DESCRIPTION
**Pull request by xhaihao**
This test case was added on the removed libva staging branch a few year ago, I added it back to libva-utils

### CODE REVIEW

File: videoprocess/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.
+# IN NO EVENT SHALL PRECISION INSIGHT AND/OR ITS SUPPLIERS BE LIABLE FOR
+# ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+noinst_PROGRAMS = videoprocess
```
Comment (uartie) [inline videoprocess/Makefile.am]: Can you add this to bin_PROGRAMS instead?

Comment (xhaihao) [inline videoprocess/Makefile.am]: @uartie I'd like not to install too many test tools in the system path, how do you think?

Comment (uartie) [inline videoprocess/Makefile.am]: If it is not installed then it makes it hard for QA test scripts to use them without knowing where libva source is compiled.

Could we just start adding a friendlier prefix to the binary names to distinguish them as a libva test/validation utility.  Perhaps va-videoprocess or va-util-videoprocess or something?

Comment (uartie) [inline videoprocess/Makefile.am]: ... or vavpp

Comment (uartie) [inline videoprocess/Makefile.am]: Also, perhaps just make it an optional install via configure flag that is off by default?

**Comment by xhaihao** [comment]
@uartie I changed the binary program name to vavpp and replaced noinst_PROGRAMS with bin_PROGRAMS. Could you please review again?

**Pull request merged**